crowdin.yaml

```diff
@@ -0,0 +1,49 @@
+project_identifier_env: CROWDIN_DOCUSAURUS_PROJECT_ID
+api_key_env: CROWDIN_DOCUSAURUS_API_KEY
+base_path: "./"
+preserve_hierarchy: true
+
+files:
+  -
+    source: '/docs/**/*.md'
+    translation: '/website/translated_docs/%locale%/**/%original_file_name%'
+    languages_mapping: &anchor
+      locale:
+        'af': 'af'
+        'ar': 'ar'
+        'bs-BA': 'bs-BA'
+        'ca': 'ca'
+        'cs': 'cs'
+        'da': 'da'
+        'de': 'de'
+        'el': 'el'
+        'es-ES': 'es-ES'
+        'fa': 'fa-IR'
+        'fi': 'fi'
+        'fr': 'fr'
+        'he': 'he'
+        'hu': 'hu'
```

Comment:
Does having _all_ of these languages mean that users will actually see that many choices? If so, we should slim it down to just English, Chinese, and any language we actually have significant content in.

Reply:
This file is generated when running script ``, the flag to control whether the user can see the lang in drop down is controlled [here](https://github.com/substrate-developer-hub/docs/pull/93/files#diff-8690ce0767c84cc3ba0f00c5d757cf9dR10). 
Currently, I only enabled the English and Chinese.